src/scss/_mixins.scss

```diff
@@ -58,7 +58,7 @@
 /// @param {String|Number} $size
 @mixin oLoadingSize($size) {
 	@if type-of($size) == 'string' {
-		$size: map-get($o-loading-sizes, $size);
+		$size: indexof($o-loading-sizes, $size);
```

Comment:
Shouldn't this stay as `map-get`? My interpretation is that if `$size` is a number it represents a pixel value and shouldn't get from the `$o-loading-sizes` map.